server/src/test/java/org/elasticsearch/rest/RestResponseTests.java

```diff
@@ -114,11 +113,10 @@ public void testSimpleExceptionMessage() throws Exception {
         Exception t = new ElasticsearchException("an error occurred reading data", new FileNotFoundException("/foo/bar"));
         RestResponse response = new RestResponse(channel, t);
         String text = response.content().utf8ToString();
-        assertThat(text, containsString("ElasticsearchException[an error occurred reading data]"));
+        assertThat(text, containsString("an error occurred reading data"));
```

Comment:
Nit: wouldn't it better if this test is changed to be more similar to testDetailedExceptionMessage, asserting we have "type" and "reason" fields?

Reply:
Yes, good spot